client/pages/applications/[id].tsx

```diff
@@ -114,7 +116,8 @@ const ApplicationDetails: NextPage = () => {
                     Download As PDF
                   </Button>
                 </div>
-              ) : (
+              )}
+              {applicationStatus === ApplicationStatus.WORKSHOP && !user?.isAdmin ? null : (
```

Comment:
```
{(applicationStatus === ApplicationStatus.WORKSHOP && !user?.isAdmin) && (
  ...
)}
```

Reply:
Should it be `user?.isAdmin`?

Reply:
this would show if an application is in the Approved or  denied status